.github/chainguard/self.release.prepare-release.sts.yaml

```diff
@@ -0,0 +1,13 @@
+issuer: https://token.actions.githubusercontent.com
+
+subject: repo:DataDog/terraform-provider-datadog:ref:refs/heads/master
+
+claim_pattern:
+  event_name: workflow_dispatch
+  job_workflow_ref: DataDog/terraform-provider-datadog/\.github/workflows/prepare_release\.yml@.*
+  ref: refs/heads/master
+  ref_protected: "true"
```

Comment:
I think this workflow always runs on `release/*` branches, so all claims will need to be adjust to that. The team should add branch protection to `release/*` branches to restrict access to those branches